README.md

```diff
@@ -69,3 +69,18 @@ $ ./target/release/cennznet --dev
 ## Contributing
 
 All PRs are welcome! Please follow our contributing guidelines [here](docs/CONTRIBUTING.md).
+
+## Community
+
+Join our official CENNZnet Discord server 🤗
+
+* Get CENNZnet technical support 🛠
+* Meet startups and DApp developers 👯‍♂️
+* Learn more about CENNZnet and blockchain 🙌
+* Get updates on CENNZnet bounties and grants 💰
+* Hear about the latest hackathons, meetups and more 👩‍💻
+
+
+Join the Discord server by clicking on the badge below!
+
+[![Support Server](https://img.shields.io/discord/591914197219016707.svg?label=Discord&logo=Discord&colorB=7289da&style=for-the-badge)](https://discord.gg/AnB3tRtkJ4)
```

Comment:
I think the badge needs to be updated to point to our server ID
it displays like this, so guess it needs to be enabled on discord server also
<img width="275" alt="Screen Shot 2021-02-16 at 15 26 34" src="https://user-images.githubusercontent.com/5133901/108011150-5bddb480-706b-11eb-9e20-afc0ccea9161.png">

```suggestion
[![Support Server](https://img.shields.io/discord/801219591636254770.svg?label=Discord&logo=Discord&colorB=7289da&style=for-the-badge)](https://discord.gg/AnB3tRtkJ4)
```

Reply:
Oh true yes haha thanks! I'll make the change and enable it